taskotron_python_versions/naming_scheme.py

```diff
@@ -1,21 +1,53 @@
+import collections
+
+from .common import log, BUG_URL
+
+
+INFO_URL = (
+    'https://python-rpm-porting.readthedocs.io/en/latest/naming-scheme.html')
+
+TEMPLATE = """These RPMs' names violate the new Python package naming guidelines:
```

Comment:
As i see now, the template seems fairly similar tot he two_three one. What if we only define the first line and have the template in common?